### ISSUE TITLE
Property 'presentable' does not exist on type 'SchemaField'

### ISSUE THREAD

**Issue by hungcrush** [pocketbase/js-sdk#258]
Sorry, should we add `presentable` to SchemaField ?

**Comment by ganigeorgiev** [comment]
Yes, I forgot to add it. I'll update it with the next release (the related Dart DTO will also have to be changed).

As a temp workaround for now if you need to access it or set it in a TS environment you can try using the brackets-string notation, aka. `field["presentable"] = true`.
